test/perf/shootout/k_nucleotide.jl

```diff
@@ -46,7 +46,7 @@ function sorted_array(m::Dict{AbstractString, Int})
         kn[i] = KNuc(elem...)
         i += 1
     end
-    sort!(kn)
+    sort(kn)
```

Comment:
It's fine to use `sort!` here.  It just means that there'll be a small bump in performance which is only due to changes in the benchmark, not changes in Julia.  If one were interested in, say, tracking Julia over time (e.g., like speed.julialang.org used to), this might be important.

> We use `f.()` in the Laplace benchmark now, how is using inplace APIs where available any different than that?

This syntax wasn't previously available in Julia, so the (presumed) speedup is precisely because of the syntax change.

Anyway, I am (and was) nitpicking.  In some circumstances, benchmark changes really should be avoided, but for Julia, it should be easy to point to the blip in the k_nucleotide timings and note that it was caused by a change in the test.

And if it's more important to compare with other languages, by all means, this should use `sort!`. 

Cheers!